collections/Lat_th/MS_Lat_th_d_13.xml

```diff
@@ -49,151 +42,147 @@
                      <idno type="SCN">Not in SC (late accession)</idno>
                   </altIdentifier>
                </msIdentifier>
-               <head>
-                     <title>Collection of Carthusian devotional works; eight MSS, c. 1460-79; German, Erfurt</title>
-                  </head>
-               
+               <head>Works of <persName key="person_88951828">Ioannes Hagen de Indagine</persName> and others</head>
                <physDesc>
-                  
-                     
-                     
-                        <p n="composite">eight parts</p>
                   <objectDesc form="codex">
                      <supportDesc material="chart">
-                        <extent>369 leaves<dimensions type="leaf" unit="mm">
-                              <height>220</height>
-                              <width>150</width>
-                           </dimensions>
-                        </extent>
+                        <support>paper; fabric tab dividers at the opening of each work</support>
+                        <extent>ii (modern endleaves) + 366 (foliated 3–368) + i (modern endleaf) <dimensions type="leaf" unit="mm">
+                              <height>216</height>
+                              <width>157</width>
+                           </dimensions> (except for fols. 172–185)</extent>
                      </supportDesc>
                   </objectDesc>
-                  
+                  <bindingDesc>
+                     <binding notBefore="1700" notAfter="1900">
+                        <p>Pulpboard lined with marbled paper and backed with white leather, 18th or 19th century.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
-                  <provenance>'<orgName role="fmo" key="org_415145858126223022295">
-                        Cartusiae Erford
-                     </orgName>' on 1st fol. &amp; top edge, 16th century</provenance>
-                  <provenance>Bought, 1929.</provenance>
+                  <origin>
+                     <origPlace>
+                        <country key="place_7000084">German</country>, <settlement key="place_7024031">Erfurt</settlement>, <orgName role="fmo" key="org_415145858126223022295">Charterhouse of Salvatorberg</orgName></origPlace>: inscribed, <q>Ad Carthusiam Erford</q> (head fore-edge, 16th century?); <q>Cartusiæ Erford</q> (fol. 1r, 17th or 18th century). Some booklets are autographs of <persName key="person_88951828" role="fmo">Ioannes Hagen de Indagine</persName>, dated 1457, 1458, 1459, 1460, 1468. These correspond to his periods of activity at Erfurt: 1440–54, 1457–60, and 1467–1475×76. Examples of his script also appear in MS. Hamilton 46, MS. Hamilton 48, MS. Hamilton 49, MS. Hamilton 54, and MS. Hamilton 57.</origin>
```

Comment:
I think origin and provenance should be separated here - yes, written at the charterhouse (at least in part) s. xv 3/4, but then also owned there until its dissolution.